src/screens/WebsocketResearchScreen/SignedChatScreen.tsx

```diff
@@ -3,41 +3,42 @@
 /* eslint-disable import/no-unresolved */
 
 import * as React from 'react';
+import ToastMessage from 'react-native-toast-message';
 import {FlatList, View} from 'react-native';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements are not sorted alphabetically. This is not a critical issue, but it's a good practice to keep imports organized for better readability and maintainability.
```diff
- import ToastMessage from 'react-native-toast-message';
- import {FlatList, View} from 'react-native';
+ import {FlatList, View} from 'react-native';
+ import ToastMessage from 'react-native-toast-message';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->